modules/ai/assets/js/editor/hooks/use-upgrade-message.js

```diff
@@ -9,7 +9,7 @@ const useUpgradeMessage = ( { usagePercentage, hasSubscription } ) => {
 	const isBelowThreshold = usagePercentage < USAGE_PERCENTAGE_THRESHOLD;
 	const isAboveThreshold = usagePercentage >= USAGE_PERCENTAGE_THRESHOLD;
 
-	const showBadge = isAboveThreshold || ( isFreeUser && isBannerViewed );
+	const showBadge = isAboveThreshold || isFreeUser;
```

Comment:
Upgrade badge should always be shown to free users

Reply:
but if they press the close button?

Reply:
it's the badge, no close button on badge

Reply:
If you checked it then all fine, cause Its not what Im seeing..

Reply:
<img width="731" alt="image" src="https://github.com/user-attachments/assets/9647765c-93bc-41c2-8d5f-687f168ac9d3">
what are you seeing?

Reply:
yup but it is also for the upgrade message I think.. try to display the purple upgrade message and then try to close it and reopen the popup and see if it still there